### DISCUSSION TITLE
Compliance with Standard for Public Code

### DISCUSSION THREAD

**Discussion by Ainali** [publiccodeyml/publiccode.yml#136] [Ideas]
I would like to suggest a field to indicate compliance with [Standard for Public Code](https://standard.publiccode.net/). This would make it easy to discover this at a glance, and even make filtering in catalogs possible. 

I would propose to not only make it a true or false, but that it also should show if a commitment to meet the requirements in the future has been made. This sends an important signal of the intention and trajectory of the codebase.

Accompanying this, a verification URL field would be handy, to either see the compliance or the documented public commitment. This would be mandatory unless, the compliance field is set to false.

**Comment by bfabio** [comment]
@Ainali I think that's a great idea. Do you have in mind something structured for the verification URL or are you thinking of a normal web page?

**Reply by Ainali** [reply]
We are aiming for something structured so the status can be queried for by a service or shown in a badge. We discussed it in a community call, minutes here: https://blog.publiccode.net/community%20call/2021/02/11/notes-from-community-call-4-february-2021.html

It would be nice to have something human readable as well, but that may come later.

**Reply by bfabio** [reply]
@Ainali Really interesting, we can probably implement it as an optional top level key containing the URL which, if present, would indicate compliance.

Naming it can be be tricky though, because of the potential confusion between the publiccode.yml Standard and the Standard for Public Code, but maybe it's a non-issue

WDYT of `standardForPublicCodeCompliance: https://example.org/xyz`

**Reply by ericherman** [reply]
> WDYT of `standardForPublicCodeCompliance: https://example.org/xyz`

This seems fine to me.

**Comment by ruphy** [comment]
@Ainali please open a PR on this, so we can put it to votes :-)
